docs/tutorials/scala/mxnet_scala_on_intellij.md

```diff
@@ -292,11 +311,24 @@ The project's `pom.xml` will be open for editing.
 
 Click "Import Changes" in this prompt.
 
-**Step 5.** Build the project:
+**Step 5.** Setup log4j configuration
+
+Create a new file `target/classes/log4j.properties` with the contents:
```

Comment:
Can we get this added to the scala package so the build and docs build runs with less errors?

Reply:
There are log4j.properties files already in the scala package. If you want to change how much they output, that should probably be separate from this and may require some discussion.